galgebra/ga.py

```diff
@@ -526,8 +538,6 @@ def mv(self, root=None, *args, **kwargs):
         Instanciate and return a multivector for this, 'self',
         geometric algebra.
         """
-        (self.mv_I, self.mv_basis, self.mv_x) = mv.Mv.setup(ga=self)
```

Comment:
Note this is the last call-site where `Mv.setup` is used or tested - we should consider deprecating it (in another PR)